### PR TITLE
fix: btoa error

### LINKED ISSUE #113: Failed to execute 'btoa' on 'Window': The string to be encoded contains characters outside of the Latin1 range.
**Describe the bug**
Here is my test:
```js
it.only('test', () => {
    cy.matchImage();
  });
```

First run is okay, but not after then.
I think it's related to my chinese character filename.

Here is the error message:
```
Error:       InvalidCharacterError: Failed to execute 'btoa' on 'Window': The string to be encoded contains characters outside of the Latin1 range.
    at Context.eval (http://localhost:3000/__cypress/tests?p=cypress/support/e2e.js:256:76)
From previous event:
    at Context.thenFn (http://localhost:3000/__cypress/runner/cypress_runner.js:138969:63)
    at Context.then (http://localhost:3000/__cypress/runner/cypress_runner.js:139391:21)
    at wrapped (http://localhost:3000/__cypress/runner/cypress_runner.js:157145:19)
    at http://localhost:3000/__cypress/runner/cypress_runner.js:155901:15
From previous event:
    at CommandQueue.runCommand (http://localhost:3000/__cypress/runner/cypress_runner.js:155879:17)
    at next (http://localhost:3000/__cypress/runner/cypress_runner.js:156032:19)
    at http://localhost:3000/__cypress/runner/cypress_runner.js:156061:16
```
image

**To Reproduce**

**Expected behavior**

**Screenshots**

**Please complete the following information:**
 - OS and version: [e.g. Windows 10 build. 19043.1319]
 - Browser and version [e.g. chrome 22]
 - Cypress version [e.g. 8.6.0]

**Additional context**

### PR DESCRIPTION
**Pull request by iamyoki**
closes #113

### CODE REVIEW

File: src/commands.ts [hunk 1]
```diff
@@ -139,9 +139,10 @@ Cypress.Commands.add(
         if (res.error) {
           log.set(
             "message",
-            `${res.message}\n[See comparison](${LINK_PREFIX}${btoa(
-              JSON.stringify({ title, imgPath })
-            )})`
+            `${res.message}\n[See comparison](${LINK_PREFIX}${btoa(unescape(encodeURIComponent(JSON.stringify({
```
Comment (FRSgit) [inline src/commands.ts:142]: I think we shouldn't use `escape`/`unescape` methods as they're [described as deprecated](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/escape).
Is `encodeURIComponent` not enough to fix this error?

**Review by FRSgit — CHANGES_REQUESTED** [review]
Hey @iamyoki,
Thanks for your PR ❤️ I have one question regarding the implementation, but I think your approach goes in the good direction, nice job!

File: src/commands.ts (hunk 1 above)
Comment (iamyoki) [reply src/commands.ts:142]: Hi @FRSgit, thanks also for your attention, I'll look into it further soon.

Comment (iamyoki) [reply src/commands.ts]: Hi @FRSgit, after research, the best way is to implement an escape Base64, so I found this answer and followed it.
https://stackoverflow.com/a/26603875/15495687

**Review by FRSgit — APPROVED** [review]
Looks cool now ❤️  Nitpick: I've added direct link to the base64 implementation, so it's easier to be found.
Thanks for the great contribution, merging it now!

File: src/commands.ts (hunk 1 above)
Comment (FRSgit) [reply src/commands.ts]: @iamyoki Thanks for doing the research - indeed it seems that this is the best approach. Personally I would think about rereleasing the `webtoolkit.base64` on npm ([the same way `webtoolkit.md5` is released](https://www.npmjs.com/package/webtoolkit.md5)) as it's nicer to be able to share this kind of functionalities with other libraries.
But this solution is well enough for now, so I'd say let's stick with it until somebody will be willing to refactor 😄 
Thanks again!

**Pull request merged**